packages/connect/src/device/Device.ts

```diff
@@ -579,6 +583,7 @@ export class Device extends TypedEmitter<DeviceEvents> {
 
         this.name = deviceInfo.name;
 
+        // todo: move to 553
```

Comment:
What for?

Reply:
there is a block that executes only if new version differs from previous version. imho this feature falls into the category of features which may not change unless firmware version changes too 
<img width="627" alt="image" src="https://github.com/trezor/trezor-suite/assets/30367552/e0a1dc3e-6f2e-4092-bfa1-d3a058f2740e">

but I didn't want to make such unerelated runtime change here, so I put a comment.

Reply:
Ok